guides/application-logging.adoc

```diff
@@ -57,10 +57,7 @@ public class GettingStartedService {
     private static Logger log = Logger.getLogger(GettingStartedService.class.getName());
 
     public String hello(String name) {
-        if (log.isEnabled(Logger.Level.TRACE)) {
-            log.trace("called method with: " + name);
-        }
-
+        log.trace("called method with: {}", name);
```

Comment:
This isn't quite correct. For a format like this it needs to be `tracev`. However, I'd suggest using:
```java
log.tracef("called method: %s", name);
```